app/renderer/abr-document.js

```diff
@@ -350,6 +366,65 @@ AbrDocument.prototype = {
         return this.save(path, callback);
     },
 
+    initWatcher: function () {
+        var that = this;
+        var updateTitle = function () {
```

Comment:
> Technically, since the document didn't change in CodeMirror, calling updateWindowTitle() here would not add the * symbol in the window title. I didn't try it though, so I might be wrong.

If you call `setDirty()` before `updateWindowTitle()` this should work. 

> I just wanted a visual feedback for the user without forcing CodeMirror to change its state

CodeMirror state won't change. Actually `isClean()` is just asking CodeMirror if `latestGeneration` matches the current state, so if you set `latestGeneration` to -1 this would do the job.

If you don't mind I'd prefer to keep this function (update title) unique, so if we need to change it in the future we won't forget anything.